charts/posthog/values.yaml

```diff
@@ -13,7 +13,7 @@ image:
   # -- Image pull policy
   pullPolicy: IfNotPresent
 
-# -- Cloud service being deployed on. Either `gcp` or `aws`
+# -- Cloud service being deployed on. Only `gcp` is supported at the moment.
```

Comment:
That's not really accurate. We do support `aws` in the chart - but we won't have setup instructions, etc on here. 

Let's keep this as-is. README.md is a better medium for explaining supported clouds.